src/app/pages/meditationPlayer/MeditationPlayer.tsx

```diff
@@ -32,9 +33,19 @@ export default function MeditationPlayer({
     isPlaying ? '' : playGong();
   }
 
+  function endMeditation() {
+    const oldSessions = JSON.parse(localStorage.getItem('sessions') || '');
+    const today = new Date();
+    const addCurrentSession = { h: hours, m: minutes, date: today };
+    const allSessions = [...oldSessions, addCurrentSession];
+    setOver(true),
+      playGong(),
+      localStorage.setItem('sessions', JSON.stringify(allSessions));
```

Comment:
Is prettier running? This seems to not have been prettified properly

Reply:
yes, it is. And you know, what's strange?! These formatting actually happened once I save the code. Anyway, I catch the problem and will investigate.